RLTest/loader.py

```diff
@@ -8,39 +8,64 @@
 class TestFunction(object):
     is_class = False
 
-    def __init__(self, target, name=None, is_method=False):
-        self.target = target
-        if not name:
-            name = target.__name__
-
-        self.name = '{}:{}'.format(target.__module__, name)
-        self.is_method = is_method
+    def __init__(self, filename, symbol, modulename):
+        self.filename = filename
+        self.symbol = symbol
+        self.modulename = modulename
+        self.is_method = False
+        self.name = '{}:{}'.format(self.modulename, symbol)
+
+    def initialize(self):
+        module_file = open(self.filename, 'r')
+        module = imp.load_module(self.modulename, module_file, self.filename, ('.py', 'r', imp.PY_SOURCE))
+        obj = getattr(module, self.symbol)
+        self.target = obj
 
     def shortname(self):
         return self.target.__name__
 
+class TestMethod(object):
+    is_class = False
```

Comment:
Shouldn't this be in caps since it's used? At this level IS_CLASS = False

Reply:
Kept it the same as it is on TestClass and TestFunction, other parts in the code seems to use it so I did not want to change it.